modi/module/setup_module/network.py

```diff
@@ -74,7 +78,11 @@ def take_picture(self):
         self._set_property(self.CommandType.CAMERA, 100)
 
     def buzzer_on(self):
+        if self.__buzzer_flag:
```

Comment:
When using pymodi with MODI Play,
MODI Play buzzer will not start unless we call buzzer_off first